setup.py

```diff
@@ -42,18 +42,15 @@
     install_requires=[
         "plone.api",
         "plone.portlets",
-        "plone.restapi",
-        "Products.GenericSetup>=1.8.2",
+        "plone.restapi>=7.5.0",
```

Comment:
@ericof can we add a note here why 7.5.0 is necessary and add a changelog entry for that change?

Reply:
@tisto Done!

Reply:
@ericof merged. I meant something like this: 

https://github.com/kitconcept/kitconcept.contentcreator/commit/0d6279c23a4dc8e20e305ed2fbb21350f77b1873

:)